modules/learn/pages/clusters-and-availability/groups.adoc

```diff
@@ -201,9 +212,4 @@ To enable the failover of all nodes in a group:
 * With CLI, see xref:cli:cbcli/couchbase-cli-setting-autofailover.adoc[setting-autofailover].
 * With the REST API, see xref:rest-api:rest-cluster-autofailover-enable.adoc[Enabling and Disabling Auto-Failover].
 
-Note that in 7.1+, _automatic failover_ can fail over more than three nodes concurrently: this has permitted the removal of pre-7.1 interfaces that were specific to triggering _auto-failover for server groups_.
```

Comment:
We need a modified version of this paragraph in lines 204-205. I realize I had previously said that they should be removed, but looking at the information in this section again, I see that it's needed.  Also, users may navigate to this section from the right-hand menu without looking at the rest of the page.  

The paragraph should be the same one from the top of the page:

Note that In 7.1+, _automatic failover_ can fail over more than three nodes concurrently: this has permitted the removal of pre-7.1 interfaces that were specific to triggering _auto-failover for server groups_. 
Consequently, for auto-failover of a server group to be possible, the maximum count for auto-failover must be established by the administrator as a value equal to or greater than the number of nodes in the server group.  Note that for auto-failover to occur, all of the auto-failover constraints listed in xref:./automatic-failover.adoc#auto-failover-constraints[Auto-failover Constraints] must be met, including the majority quorum requirement -- that the remaining nodes must be able to form a majority quorum to be able to initiate an auto-failover. If, for example, you have two server groups, with equal number of nodes in each, and if all of the nodes in one of the server groups fail at the same time, even if you have the maximum count for auto-failover set to a value equal to or greater than the number of nodes in the server group that failed, auto-failover cannot occur since the remaining nodes in the remaining server group cannot form a majority quorum.